src/Panel.js

```diff
@@ -23,6 +25,18 @@ const Panel = React.createClass({
     };
   },
 
+  getInitialState(){
+    let defaultExpanded = this.props.defaultExpanded != null ?
+      this.props.defaultExpanded :
+        this.props.expanded != null ?
+        this.props.expanded :
```

Comment:
It would be an anti-pattern if your component doesn't need to change the value, which is not the case here, no?